docs/guides/Integration-Guides/sent-offers-integration-guide-connectors/offer-management-sdk.md

```diff
@@ -5,43 +5,24 @@ hidden: false
 createdAt: "2022-07-06T14:04:18.259Z"
 updatedAt: "2022-07-06T14:38:05.752Z"
 ---
-[block:callout]
-{
-  "type": "info",
-  "body": "To evolve your experience in synchronizing and updating offers, the VTEX team is developing a new functionality that will replace the current module **Offer Management**. As a result, the module will not receive new updates or maintenance.
+>⚠️To evolve your experience in synchronizing and updating offers, the VTEX team is developing a new functionality that will replace the current module **Offer Management**. As a result, the module will not receive new updates or maintenance.
```

Comment:
É preciso adicionar um espaço entre o emoji e o início da frase, como visto [aqui](https://github.com/vtexdocs/dev-portal-content#how-can-i-add-different-colored-callouts)